.github/workflows/build.yml

```diff
@@ -226,10 +222,24 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Process user data file
+        id: process-user-data
+        # Process the user data script with the Windows admin password
+        # pulled from AWS Parameter Store. Return the admin password to
+        # pass into Packer as an environment variable.
+        run: |
+          output=$(python .github/scripts/process_user_data_file.py)
+          echo "::add-mask::$output"
+          echo "::set-output name=pass::$output"
       - name: Create machine image
         # This runs through the AMI creation process but does not
-        # actually create an AMI
+        # actually create an AMI.
+        # See https://www.packer.io/guides/hcl/variables under the
+        # "From environment variables" section for details about
+        # PKR_VAR environment variables
         run: |
+          PKR_VAR_admin_pass=\
```

Comment:
What is the rationale for using an environment variable instead of passing the value through the same way we do for other variables? You appear to be masking it already in the above `process-user-data` step.

Reply:
I couldn't really find the answer to this but i previously tried to pass it as a regular var before with no success. I just tried it again with this commit: [062ed6c](https://github.com/cisagov/windows-server-packer/pull/10/commits/062ed6c496952c282f87454f734de82bea2b33ef)

I'm not getting a useful error in gh actions either. But that's pretty much why i resorted to using an env var. I also remember reading for automated processes, I should be passing secrets via env vars this way. But never verified if a regular var flag would work or not.

Reply:
reverted with: [83f7f2f](https://github.com/cisagov/windows-server-packer/pull/10/commits/83f7f2f9145d74921a87df7cfdcdb9e0b5cf5990)